### PR TITLE
Fixing RBAC roles

### PR DESCRIPTION
**Pull request by ahg-g**
#### What type of PR is this?

/kind bug

### CODE REVIEW

File: config/rbac/role.yaml [hunk 1]
```diff
@@ -44,25 +52,17 @@ rules:
...
 - apiGroups:
   - kueue.x-k8s.io
   resources:
-  - clusterQueues/status
+  - clusterqueues/status
   verbs:
   - get
   - patch
   - update
-- apiGroups:
```
Comment (ArangoGutierrez) [inline config/rbac/role.yaml:58]: revert?

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

**Comment by alculquicondor** [comment]
Weird... don't we verify this in pull-kueue-verify-main?

**Comment by alculquicondor** [comment]
nvm, the tags themselves were wrong.

**Pull request merged**